js/tests/unit/collapse.js

```diff
@@ -52,6 +52,16 @@ $(function () {
     assert.ok(!/height/i.test($el.attr('style')), 'has height reset')
   })
 
+  QUnit.test('should collapse only the first collapse', function (assert) {
```

Comment:
The collapse that you toggle needs to be within an accordion, so that it has a parent and thus triggers the code path that you've changed.